hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/proxy/ViewExecutor.java

```diff
@@ -166,6 +168,25 @@ private Bytes answerGiven(final int selector) {
                         ResponseCodeEnum.INVALID_TOKEN_ID);
                 final var token = stateView.tokens().get(EntityNum.fromTokenId(wrapper.tokenID()));
                 return encoder.encodeGetTokenType(token.tokenType().ordinal());
+            }
+            case ABI_ID_GET_TOKEN_EXPIRY_INFO -> {
+                final var wrapper = decoder.decodeGetTokenExpiryInfo(input);
+                validateTrueOrRevert(
+                        stateView.tokenExists(wrapper.tokenID()),
+                        ResponseCodeEnum.INVALID_TOKEN_ID);
+                final var tokenInfo = stateView.infoForToken(wrapper.tokenID()).orElse(null);
+
+                if (tokenInfo == null) {
```

Comment:
You can use `validateTrueOrRevert` check.

Reply:
Same as above comment.